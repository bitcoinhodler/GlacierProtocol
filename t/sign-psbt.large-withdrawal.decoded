{
  "txid": "7bca0c14a9d3565ade5cecdc849bce4020c445099581bae551d8c90ae6220ad9",
  "hash": "7bca0c14a9d3565ade5cecdc849bce4020c445099581bae551d8c90ae6220ad9",
  "version": 2,
  "size": 3247,
  "vsize": 3247,
  "weight": 12988,
  "locktime": 0,
  "vin": [
    {
      "txid": "51c24cb94c3985c99fa763eacc41dae3d13945acef904b4e65eafe0f7fe02188",
      "vout": 1,
      "scriptSig": {
        "asm": "0 3044022000d31a31e181334bcca372daf3721340bceb9af003dd02d7a1f143f747364aec02201d36c8863c319b1fc233ba2d9214990f115098400f66a1f9abfb9697b5b77e7a[ALL] 304402207adff1c057151377e3468f6cba9c6af892f0af27522a845b4307fb3e30a6568402205f44f8f9a9848d1a78849816b500826329350f4613850b98a4001e84aa974958[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "00473044022000d31a31e181334bcca372daf3721340bceb9af003dd02d7a1f143f747364aec02201d36c8863c319b1fc233ba2d9214990f115098400f66a1f9abfb9697b5b77e7a0147304402207adff1c057151377e3468f6cba9c6af892f0af27522a845b4307fb3e30a6568402205f44f8f9a9848d1a78849816b500826329350f4613850b98a4001e84aa974958014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    },
    {
      "txid": "d0f89f12109ee7581496279922c6cb92c74a761392e6cf70d16f6857400a1ba0",
      "vout": 1,
      "scriptSig": {
        "asm": "0 304402204ac2cc2c32dde85bdd11fe5db9c8d648a3c61debcec62b9ae5261afc1b32b08802200e6314cf0b379e3d4add8bea9d4c1383332a79afe42458130ffe277ef30a56d0[ALL] 304402206a0abd7e2c08dcb75d43563669d0b093e551f75b2cb64476700b849a4b510ace022073f317cde04e533aaea7f93488b26511b6c886c649f7c3bd28d1be42a5a18088[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "0047304402204ac2cc2c32dde85bdd11fe5db9c8d648a3c61debcec62b9ae5261afc1b32b08802200e6314cf0b379e3d4add8bea9d4c1383332a79afe42458130ffe277ef30a56d00147304402206a0abd7e2c08dcb75d43563669d0b093e551f75b2cb64476700b849a4b510ace022073f317cde04e533aaea7f93488b26511b6c886c649f7c3bd28d1be42a5a18088014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    },
    {
      "txid": "c50c0b65ec506ed34552666dbb931791446471443a79e29a30c41a5b1cc20a4f",
      "vout": 1,
      "scriptSig": {
        "asm": "0 3044022040f49ba184328568e1c1332dd25b63278faef847a4382afa686601ea8552a56d0220273c40f89d166ee17ca9b401faec18886fedf4134d4a21c08d9a0bd1166167f3[ALL] 304402207487b8a210d4c915b5b1227da684c7cc9d7e11c5052b133aae2a0587cff71f8d0220180fbc454847dca846583551c32d90314f6e5dc6bb03697f28b878dc3c7786c2[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "00473044022040f49ba184328568e1c1332dd25b63278faef847a4382afa686601ea8552a56d0220273c40f89d166ee17ca9b401faec18886fedf4134d4a21c08d9a0bd1166167f30147304402207487b8a210d4c915b5b1227da684c7cc9d7e11c5052b133aae2a0587cff71f8d0220180fbc454847dca846583551c32d90314f6e5dc6bb03697f28b878dc3c7786c2014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    },
    {
      "txid": "fe20dad1ac9bc35fa7cb205c7f672f89a0b72dfbb4e20a9741adc47d89a83ec8",
      "vout": 0,
      "scriptSig": {
        "asm": "0 304402207bd47296fcf655f92cfdd035f809b4aba56c4675e46969243be71bc12a4f39690220557a92a4fa5d9b140617fb6d3399cf8c8c3186e474651a606be472ee81d7ea19[ALL] 304402204d770b9f3bad99133a5d04aa61185d1567eca5925aab2ec7f31997af409dcfaf02207694cc0f0b31b7438ac489223ddb0ceaa5cef18494465dc50126ef1e719b71f7[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "0047304402207bd47296fcf655f92cfdd035f809b4aba56c4675e46969243be71bc12a4f39690220557a92a4fa5d9b140617fb6d3399cf8c8c3186e474651a606be472ee81d7ea190147304402204d770b9f3bad99133a5d04aa61185d1567eca5925aab2ec7f31997af409dcfaf02207694cc0f0b31b7438ac489223ddb0ceaa5cef18494465dc50126ef1e719b71f7014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    },
    {
      "txid": "fb129bea8fbefaefa816c6b722cffc4b89691b78a8a9a6d5b1aecb6d7918347b",
      "vout": 1,
      "scriptSig": {
        "asm": "0 3044022018771137d22339b0cc33e844fb7ddfeedd3bab360ebca708d3727860803b9fbc02202ea79237e8f13cbee9219f8911b28daf9a3fea936729da524649f3133fd6015d[ALL] 304402207c69038c187f7a6becf5f3d309859b1c95559fdbe0ab8a06e497765578c1839202206be3ed0cecb15f1cb2368167db8dc4451b7f69633c47c8c6635d533659dbd222[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "00473044022018771137d22339b0cc33e844fb7ddfeedd3bab360ebca708d3727860803b9fbc02202ea79237e8f13cbee9219f8911b28daf9a3fea936729da524649f3133fd6015d0147304402207c69038c187f7a6becf5f3d309859b1c95559fdbe0ab8a06e497765578c1839202206be3ed0cecb15f1cb2368167db8dc4451b7f69633c47c8c6635d533659dbd222014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    },
    {
      "txid": "59f199ff6f1c23ebfe3c3f6f369b7e468dc358f483b92f3b9507cdb07385ba08",
      "vout": 0,
      "scriptSig": {
        "asm": "0 304402200a84c21cb078503a0608c83c07808d1ce9fc8de947b812de7801f6a0f509814d0220252d52413cc38e500fac956c931db22543cbe059209e0135ba7a3f5cd840007d[ALL] 304402206cc27f0fc960a9ad11e49682665419927e1615bd53a98e61b6d74eb1dbe2807302204a52777b06b66f8db0f1d63af8dc00c7dd768124b8b38c638353e8d01f5b9f00[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "0047304402200a84c21cb078503a0608c83c07808d1ce9fc8de947b812de7801f6a0f509814d0220252d52413cc38e500fac956c931db22543cbe059209e0135ba7a3f5cd840007d0147304402206cc27f0fc960a9ad11e49682665419927e1615bd53a98e61b6d74eb1dbe2807302204a52777b06b66f8db0f1d63af8dc00c7dd768124b8b38c638353e8d01f5b9f00014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    },
    {
      "txid": "57623b48be1cd94bbec44b186c80dc3701f2eb9639aa7ae76b7e16d87512992a",
      "vout": 0,
      "scriptSig": {
        "asm": "0 3044022020a50f92fdaf00d17d42d1315b768470384043018c11ffde40e648ead858e5b20220418dd023a426c704d20cee284b0237a172c4d08b97a3c4f32fae409c93db01b8[ALL] 30440220441386740f7f57701ad1d681c78f24784a27ec7383097f39646774f238a521cb02203dfa63e4643cf6facbb625d63872dcdc85cc130d978ee0646007983f4111ff50[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "00473044022020a50f92fdaf00d17d42d1315b768470384043018c11ffde40e648ead858e5b20220418dd023a426c704d20cee284b0237a172c4d08b97a3c4f32fae409c93db01b8014730440220441386740f7f57701ad1d681c78f24784a27ec7383097f39646774f238a521cb02203dfa63e4643cf6facbb625d63872dcdc85cc130d978ee0646007983f4111ff50014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    }
  ],
  "vout": [
    {
      "value": 0.07226753,
      "n": 0,
      "scriptPubKey": {
        "asm": "0 1b8dffa18bc3e70d2aadf872d396b6b241a79e5b",
        "desc": "addr(bcrt1qrwxllgvtc0ns624dlped894kkfq608jmye2k50)#hfrytdh0",
        "hex": "00141b8dffa18bc3e70d2aadf872d396b6b241a79e5b",
        "address": "bcrt1qrwxllgvtc0ns624dlped894kkfq608jmye2k50",
        "type": "witness_v0_keyhash"
      }
    }
  ]
}
