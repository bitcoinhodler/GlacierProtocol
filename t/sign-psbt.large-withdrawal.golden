Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?Input a filename located in the current directory which contains the PSBT:
0.07230000 BTC in unspent inputs from cold storage address 2MyCc3oEVwSYzgmspKTjZ6HZrzeg8ajdmKR
0.07226753 BTC going to destination address bcrt1qrwxllgvtc0ns624dlped894kkfq608jmye2k50
Fee amount: 0.00003247
Confirm? (y/n): 
How many private keys will you be signing this transaction with (at least 2 required to complete transaction)?
#: Key #1: Key #2: 
Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.07230000 BTC in unspent inputs from cold storage address 2MyCc3oEVwSYzgmspKTjZ6HZrzeg8ajdmKR
0.07226753 BTC going to destination address bcrt1qrwxllgvtc0ns624dlped894kkfq608jmye2k50
Fee amount: 0.00003247

Signing with private keys: 
93VDtjySSuPjgLYyCR6BBTqCuaqu6X728ZfkNEDbgAdVURRd1rS
924WjH7yMYyuyhUougBHEu2vThcHRmahF6FsKvNVQBwSGuEyqKm


Confirm? (y/n): 
Calculating transaction...

Final fee rate: 1 satoshis per vbyte

Raw signed transaction (hex):
020000000721de407a2df54b182b6e935a753228632e72727434f8ddc62a906bc3c4a80fa001000000fd9f010047304402204100064b0015c07fb3c2b7b126f016752cb1c34b16a82d1e185f2434d4ff21b3022079be36a11e9a03987995570e622bfbb1252e74984900d82c8715f1a36c9a16ee014730440220477ac40bf11a386f73713914cccacf6304e9ed155d1c5d4e50c4fffb6d326a1e02207d2ab1e4307bc85aa4cc4da6efcbe1ffd637748a733770daacb063372a3199e6014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffff7ea98084a14b86fa2338f7bf2cacb2277eded3dc87e05777339947a4edc6ff6101000000fd9f010047304402202d35c4bad270dbbe2f30900d2b3e97df05c5fd45133557baa2f3f19527b8d45f02205ce68b66774cc234b78ddc87d697920520c7b815c44b6b8c94c6a5ec16e189b10147304402205f4b4e02bbcafde5db50aa4ba67fec5cfff277635d374dc354280962ccb839c002201f6b1bf3aa9f10c6c13be116a760d5af4ee332f9aed7ec1e8202dca8ca9883e5014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffffe99cbc7cc9aefa904880d78026456847e149528cf1b319718acd9cfb14cbb32901000000fd9f0100473044022015141f672cb5a430898087635e3e80486a09aa51a6bbb2309101d7fdc116176c022004e4eb39e4caf7558f0e7f30bc9188938761b062126d7de06c661e1fb044e16d0147304402201beb617dab9c26de651379fd019bb21c816b297a3a3fc8c06bf496a0d07fd35c0220114afe0c115207aeb688e60863671cc3b0a3d4bb5bbc9cd89e0230c953457cb5014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffff5d270c99527c1c0a307e58e9a1bb02b69cce6e2790185cd78586b7c6214703ce00000000fd9f010047304402207f69b2cbd60f8fc99df1263cac8ffcb0a3a1743aa7ad20cf3311691b41aedb3e02207f3fd31539d4d513be356704a03e50face33ac2991b73ef2aa034f4100d8d69b01473044022055056b70e3794a490b1ffcf5bc82eb434f44c46fa620cc4fd2b6a9eda7a8911a0220298126773f5864d6672647588853a5975939255111f7c7f034c4ed3eca2d14ae014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffffd1c09c97e88aa79334e762745edf4dd0719e602c8d61746d5ce605fdabd32a4101000000fd9f0100473044022067c3611e30f6548f753e01eb1274e0678898f82debfed8eec384a3d16eea6f2f02204e05664329140c694271b00cc3424ceb3b48afccb19c08a178ff6b7d76369afd014730440220252e62e6b4babddd9e66b7bb8ef6fe2ef01a4a2ff729110eb97ab8a7bb45ffaf02203cc086c2c73cc02dac8813f498337de78b34b997ff51310bea40bc288c74419d014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffffa6735878b99084c792df6f7ac3959154a5e6dba740a169d56be6c25538eb4b2c00000000fd9f0100473044022039d432eba2b4610c0ac578279d891d0499f11781ec07a75846ccbf0e95bc20f602201e887a20a4fedf2132fc21e8d86d1db4ecbe282644bad88456369be10bfd79b7014730440220117a78413b6320f5f01f520bdc5bb20437d868f06fa22c8439d9b46925c3ebd402205e44597805fec4c33addc1206fab208d34e3fa9d5c13b8bf5417ef2547607a79014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffffd1d126103dcd85d971b39d9243b466a006a596b13450c262c37e91ea7c89ace700000000fd9f010047304402200c3c71625681f21489c91b9bb22de4278c5aa513d81f6790d1c6545139cf9fbd0220057c969e356216b351d90dcc548efb00a8b77f49118acdef2eab75319c2f27fd0147304402205cd931cbf3537857e15526b35072eb3fabb818daaa192cc9e82c6c7c2d109d380220021d16a4a781b7abf3c817f80e9a8736bec3c2f955b079e14a1c9c7fc2f2f717014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffff0181456e00000000001600141b8dffa18bc3e70d2aadf872d396b6b241a79e5b00000000

Transaction fingerprint (md5):
d126a6574305106b44e55d5a507e51c6
QR code for transaction written to transaction-01.png,transaction-02.png
